src/SignalR/clients/java/signalr/test/signalr.client.java.Tests.javaproj

```diff
@@ -2,75 +2,42 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
 
   <PropertyGroup>
-
-    <!-- This package ID is only ever used along with eng/PatchConfig.props to determine when to patch the Java client. -->
-    <PackageId>java:signalr</PackageId>
-
-    <!-- In servicing builds, this will be set to value if the Java client is not configured to be released in the currently building patch. -->
-    <IsPackable>true</IsPackable>
-
+    <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
-
     <!-- Installing Java on ARM will take some work -->
     <SkipHelixArm>true</SkipHelixArm>
-
     <!-- Disable gradle daemon on CI since the CI seems to try to wait for the daemon to shut down, which it doesn't do :) -->
     <GradleOptions Condition="'$(ContinuousIntegrationBuild)' == 'true'">$(GradleOptions) -Dorg.gradle.daemon=false</GradleOptions>
     <PublishDir>$(OutputPath)</PublishDir>
     <TestDependsOnJava>true</TestDependsOnJava>
-
   </PropertyGroup>
 
-  <ItemGroup>
-    <Jars Include="
-        signalr-$(PackageVersion).jar;
-        signalr-$(PackageVersion)-javadoc.jar;
-        signalr-$(PackageVersion)-sources.jar;" />
-    <PomFile Include="signalr-$(PackageVersion).pom" />
-    <JavaBuildFiles Include="@(Jars);@(PomFile)"/>
-  </ItemGroup>
-
   <Target Name="Restore" />
 
-  <PropertyGroup>
-    <PackDependsOn Condition=" '$(NoBuild)' != 'true' ">
-      $(PackDependsOn);
-      Build
-    </PackDependsOn>
-  </PropertyGroup>
-
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.targets))\Directory.Build.targets" />
 
   <!-- Define Target overrides after importing Directory.Build.targets so these don't get overridden -->
-  <Target Name="Pack" DependsOnTargets="$(PackDependsOn)" Condition="'$(IsPackable)' == 'true'">
-    <Telemetry EventName="NETCORE_ENGINEERING_TELEMETRY" EventData="Category=Pack" />
-    <Message Text="> gradlew $(GradleOptions) createPackage" Importance="high" />
-    <Exec Command="./gradlew $(GradleOptions) createPackage" />
-    <Message Importance="high" Text="java:signalr -> $(PackageOutputPath)%(JavaBuildFiles.Identity)" />
-    <Copy SourceFiles="build\libs\%(JavaBuildFiles.Identity)" DestinationFolder="$(PackageOutputPath)" />
-  </Target>
+  <Target Name="Pack" />
 
-  <Target Name="Build">
-    <Telemetry EventName="NETCORE_ENGINEERING_TELEMETRY" EventData="Category=Build" />
-    <Exec Command="./gradlew $(GradleOptions) compileJava" />
-  </Target>
+  <Target Name="Build" />
```

Comment:
You either need to have project references to the signalr and messagepack javaproj's or manually build them here